src/arrays.js

```diff
@@ -1,61 +1,79 @@
+const array = ['cat', 'dog', 'elephant', 'fox'];
 const getNthElement = (index, array) => {
-  // your code here
+  return array[index % array.length];
 };
 
+
 const arrayToCSVString = array => {
   // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
   // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
   // your code here
+  console.log(array.push(element));
 };
 
 const addToArray2 = (element, array) => {
   // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
   // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
   // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
   // your code here
+  return strings.map(string => string.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
   // your code here
+  return strings.map(string => string.split('').reverse().join(''));
 };
 
 const onlyEven = numbers => {
   // your code here
+  return numbers.filter(x => x % 2 === 0);
```

Comment:
This works perfectly, the only suggestion I would have is usually we like to describe what a parameter is through it's naming, so instead of `x` here, I would probably write `number` or `num`. Just because it is slightly more descriptive, as the letter `x` could represent anything.